samza-core/src/main/scala/org/apache/samza/container/SamzaContainer.scala

```diff
@@ -568,6 +540,11 @@ object SamzaContainer extends Logging {
         new StorageConfig(config).getChangeLogDeleteRetentionsInMs,
         new SystemClock)
 
+      val tableManager = new TableManager(config)
+      tableManager.initLocalTables(taskStores.asJava)
```

Comment:
It is possible, but it would require task instance to provide public access to its stores during init phase. We are weaving stores in to table manager, which is crossing the boundary here. I think it's probably better to do this here.